src/main/clojure/clara/rules/platform.cljc

```diff
@@ -8,6 +8,22 @@
   (throw #?(:clj (IllegalArgumentException. description) :cljs (js/Error. description))))
 
 #?(:clj
+   (defn group-by-seq [f coll]
```

Comment:
Could you add a (shortened) version of the commentary on the PR description here about respecting the order in the code here?  If I were just reading the code without that context the reason to set it up this way wouldn't be obvious.